### DISCUSSION TITLE
How can I confirm the following agreement as a non-commercial user??

### DISCUSSION THREAD

**Discussion by thegr8estmay** [coqui-ai/TTS#3477] [General Q&A]
> You must confirm the following:
 | > "I have purchased a commercial license from Coqui: licensing@coqui.ai"
 | > "Otherwise, I agree to the terms of the non-commercial CPML: https://coqui.ai/cpml" - [y/n]
 | | > ?????

**Comment by Maskerss** [comment]
write y,then enter....

**Comment by mithenks** [comment]
You can define a `COQUI_TOS_AGREED` with the value `1`. 

It's possible to embed it in your python script:
```
import os
os.environ["COQUI_TOS_AGREED"] = "1"
```

**Comment by ellosykageni** [comment]
os.environ["COQUI_TOS_AGREED"] = "1"
